packages/reactive-dict/reactive-dict.js

```diff
@@ -124,6 +126,31 @@ _.extend(ReactiveDict.prototype, {
     if (_.has(self.keys, key)) oldValue = parse(self.keys[key]);
     return EJSON.equals(oldValue, value);
   },
+  
+  all: function() {
+    this.allDeps.depend();
+    var ret = {};
+    _.each(this.keys, function(value, key) {
+      ret[key] = parse(value);
+    });
+    return ret;
+  },
+  
+  clear: function() {
+    var self = this;
+    
+    var oldKeys = self.keys;
+    self.keys = {};
+    
+    self.allDeps.changed();
+    
+    _.each(oldKeys, function(value, key) {
```

Comment:
Currently, there is no way to do the things that `clear()` and `all()` let you do, so this is actually an upgrade in functionality (resetting a reactive dict and being able to iterate over its contents) rather than just API sugar.